src/test/java/org/folio/rest/api/LoansApiTest.java

```diff
@@ -1329,9 +1329,9 @@ public void cannotProvideAdditionalPropertiesInLoanStatus()
   @Test
   public void canSearchByLoanStatus() throws Exception {
     final IndividualResource openLoan = loansClient.create(
-      new LoanRequestBuilder().open().checkedOut());
+      new LoanRequestBuilder().checkedOut());
 
-    loansClient.create(new LoanRequestBuilder().closed().checkedOut());
+    loansClient.create(new LoanRequestBuilder().checkedIn());
```

Comment:
Given this search is about the loan status, rather than the item status, why has the explicit specification of a loan status been removed?

Reply:
The status has been encapsulated in the checkIn/out methods of loan builder.